_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -242,6 +242,7 @@ typedef struct tagENCODE_CAPS
             UINT    NoESS                       : 1;
             UINT    Color420Only                : 1;
             UINT    ICQBRCSupport               : 1;
+            UCHAR   BrcSlidingWindowFrameToleranceMode: 1;
```

Comment:
The rest of the patch looks good. Now we need to correctly define this variable. @stasefremov : can you, please, advise hot to define it?

Reply:
Should be either :2 or some enum. @GlaUgrub : still need to place this correctly.